### PR TITLE
Filter PII data sent to sentry by hash key

### PR DESCRIPTION
**Pull request by rlbaker**
This will filter the values of any keys before it gets sent to sentry. It handles nested dictionaries and will normalize the keys before filtering to try and catch any `snake_case`, `CameCase`, or `camelCase` fields. This is a bit lightweight at the moment, so let me know if we need more strict sanitization.

### CODE REVIEW

File: spec/lib/sentry/processor/pii_sanitizer_processor_spec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sentry/processor/email_sanitizer'
```
Comment (lihanli) [inline spec/lib/sentry/processor/pii_sanitizer_processor_spec.rb]: not needed

Comment (rlbaker) [reply spec/lib/sentry/processor/pii_sanitizer_processor_spec.rb]: Whoops. Pulled that import accidentally from the other one. Thanks.

**Comment by lihanli** [comment]
does this work for JSON strings? for VIC::SubmissionJob the form variable is a JSON string http://sentry.vetsgov-internal/vets-gov/platform-api-staging/issues/33543/

**Comment by rlbaker** [comment]
No, but I saw some example Raven processors that handle JSON blobs. I'll add that in.

File: lib/sentry/processor/pii_sanitizer.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Sentry
+  module Processor
+    class PIISanitizer < Raven::Processor
+      SANITIZED_FIELDS = %w[
```
Comment (jkassemi) [inline lib/sentry/processor/pii_sanitizer.rb]: minor nit: set vs array

**Pull request merged**